apps/demo-ui/ui/src/pages/PDFViewer/PdfViewer.tsx

```diff
@@ -0,0 +1,336 @@
+import React, { useRef } from "react";
```

Comment:
should this be called index.tsx like the others?

Reply:
Yes, i missed renaming that file. Will change it